src/datadog-lambda.ts

```diff
@@ -177,12 +176,10 @@ export function addCdkConstructVersionTag(lambdaFunction: lambda.Function): void
   });
 }
 
-function setTagsForFunctions(lambdaFunctions: lambda.Function[], props: DatadogLambdaProps): void {
-  lambdaFunctions.forEach((lambdaFunction) => {
-    if (props.forwarderArn) {
-      setTags(lambdaFunction, props);
-    }
-  });
+function setTagsForFunctions(lambdaFunction: lambda.Function, props: DatadogLambdaProps): void {
```

Comment:
nit: should we rename it to `setTagsForFunction` now that it only takes in one?

Reply:
Good catch! Will fix.